kubernetes/hmda-spark-reporting/hmda-spark-disclosure.yaml

```diff
@@ -6,12 +6,7 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  sparkConf:
-    "spark.streaming.receiver.maxRate": "1"
-    "spark.streaming.receiver.maxRate": "1"
-    "spark.streaming.backpressure.initialRate": "1"
-    "spark.streaming.backpressure.enabled": "true"
-  image: "hmda/hmda-spark-reporting:gead783482"
+  image: "hmda/hmda-spark-reporting:proddata"
```

Comment:
We should switch the image tag back before merge.

Reply:
Resolved